.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
Its actually a good idea to put this in your global gitignore, instead of the project one, as not everyone uses phpstorm